<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1916</width>
    <height>980</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Image Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/ic_photo_album_128_28567.ico</normaloff>../../Downloads/ic_photo_album_128_28567.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="menuFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>178</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QPushButton" name="openDatabaseButton">
         <property name="text">
          <string>Открыть альбом</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="createDatabaseButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Создать фотоальбом</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="tabChangesFocus">
          <bool>false</bool>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Список доступных фотоальбомов&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="deleteAlbumButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Удалить альбом</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="databaseListWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutButtons">
        <item>
         <widget class="QPushButton" name="addPhotoButton">
          <property name="text">
           <string>Добавить фото</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePhotoButton">
          <property name="text">
           <string>Удалить фото</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prevPhotoButton">
          <property name="text">
           <string>Предыдущая фотография</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextPhotoButton">
          <property name="text">
           <string>Следующая фотография</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortDownButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Сначала новые</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortUpButton">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сначала старые</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="imageLabel">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="pixmap">
       <pixmap/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1916</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Инфо</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Разработчик</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
